File: 151004/Ivanenko/DC_Lab1/DC_Lab1/DTO/PostRequestTo.cs
using DC_Lab1.DTO.Interface;

namespace DC_Lab1.DTO
{
    public class PostRequestTo(int Id,int? TweetId, string? Content) : IRequestTo
    {
        public int Id { get; set; } = Id;
        public int? TweetId { get; set; } = TweetId;

        public string? Content { get; set; } = Content;
    }
}

File: 151001/Baran/Lab5/discussion/src/main/java/by/bsuir/dto/PostRequestTo.java
package by.bsuir.dto;

import com.fasterxml.jackson.annotation.JsonInclude;
import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.Size;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@JsonInclude(JsonInclude.Include.NON_NULL)
public class PostRequestTo {
    private Long id;
    private String country;
    @NotBlank
    @Size(min = 2, max = 2048)
    private String content;
    private Long issueId;
    private String method;
}
